startup.yml

```diff
@@ -2,7 +2,7 @@ apps:
   - name: BystroAnnotationServer
     script: perl/bin/bystro-server.pl
     interpreter: "perl"
-    args: -q config/beanstalk.yml --debug
+    args: -q config/beanstalk.yml
```

Comment:
we now log all debug messages by default (and have removed the debug parameter). better to have detailed logging for diagnosing network issues.